### DISCUSSION TITLE
Bug/Feedback on pull shark badge

### DISCUSSION THREAD

**Discussion by hemantwasthere** [community/community#29404] [Other Feature Feedback, Questions, & Ideas] [answered]
I have contributed on many repos and many of mine pull requests got merged,
But its not showing x2 or x3 on pull shark achievements. 

Is this a bug or I am missing something for showing that on my pull shark achievement

**Comment by melvincwng** [comment]
@hemantwasthere Hi there, from your profile, I can see that you have 16 PRs which should qualify you for the x2 Pull Shark Achievement badge.

Here's my suggestion:
1. Wait for a couple of hours to see if your Pull Shark achievement badge get updated. There could be a delay in the updating of the achievement badges.
2. If that doesn't work, consider submitting a ticket to [GitHub Support](https://support.github.com/contact/account). Explain to them your issue that you submitted more than the required PRs, but your badge isn't updating.

Hope this clarifies.

Thanks, and take care.

**Reply by queenofcorgis** [reply]
Thanks @melvincwng for the thorough explanation !

**Reply by hemantwasthere** [reply]
Thanks @queenofcorgis now i got the issue 😌

**Comment by DenverCoder1** [comment, marked as answer]
It seems to me that only 9 of your pull requests are merged. To get the Pull Shark x2, you need a minimum of 16.

https://github.com/search?q=is%3Apr+is%3Amerged+author%3Ahemantwasthere&type=Issues

**Reply by hemantwasthere** [reply]
Thanks for your answer 😌
Now i got the answer
